esmvaltool/recipes/recipe_climwip_brunner2019_med.yml

```diff
@@ -0,0 +1,295 @@
+
+  region: &region
+    <<: *general
+    extract_shape:
+      shapefile: 'shapefiles/SREX/MED/MED.shp'  # <-----
+      method: contains
+      crop: true
+
+  region_mean:
+    <<: *region
+    <<: *mean
+
+  region_std:
+    <<: *region
+    <<: *std
+
+  temperature_anomalies:
+    custom_order: true
+    <<: *region
+    area_statistics:
+      operator: mean
+    annual_statistics:
+      operator: mean
+    anomalies:
+      period: full
+      reference:
+        start_year: 1981
+        start_month: 1
+        start_day: 1
+        end_year: 2010
+        end_month: 12
+        end_day: 31
+      standardize: false
+
+diagnostics:
+  calculate_weights_climwip:
+    variables:
+      tas_CLIM: &common_settings
+        short_name: tas
+        start_year: 1995
+        end_year: 2014
+        mip: Amon
+        preprocessor: region_mean
+        additional_datasets: *obs_data
+      pr_CLIM:
+        <<: *common_settings
+        short_name: pr
+      rsds_CLIM:
+        <<: *common_settings
+        short_name: rsds
+      rlds_STD:
+        <<: *common_settings
+        short_name: rlds
+        preprocessor: region_std
+      rsus_CLIM:
+        <<: *common_settings
+        short_name: rsus
+        derive: true
+        force_derivation: false
+      rsus_STD:
+        <<: *common_settings
+        short_name: rsus
+        preprocessor: region_std
+        derive: true
+        force_derivation: false
+      # only needed if no performance sigma is given
+      # CLIM_future:
+      #   short_name: tas
+      #   start_year: 2081
+      #   end_year: 2100
+      #   mip: Amon
+      #   preprocessor: region_mean
+
+    scripts:
+      climwip:
+        script: weighting/climwip/main.py
+        obs_data: native6
+        combine_ensemble_members: false
+        performance_sigma: 0.546  # <-----
+        independence_sigma: 0.643  # <-----
+        performance_contributions:
+          tas_CLIM: 1
+          pr_CLIM: 1
+          rsds_CLIM: 1
+          rsus_CLIM: 1
+          rlds_STD: 1
+          rsus_STD: 1
+        independence_contributions:
+          tas_CLIM: 1
+          pr_CLIM: 1
+          rsds_CLIM: 1
+          rsus_CLIM: 1
+          rlds_STD: 1
+          rsus_STD: 1
+        # only needed if no performance sigma is given
+        # calibrate_performance_sigma:
+        #   target: CLIM_future
+
+  weighted_temperature_graph:
+    variables:
+      tas:
+        start_year: 1960
+        end_year: 2100
+        mip: Amon
+        preprocessor: temperature_anomalies
+    scripts:
+      weighted_temperature_graph:
+        script: weighting/weighted_temperature_graph.py
+        ancestors: [calculate_weights_climwip/climwip, tas]
+        weights: 'weights.nc'
+
+  weighted_temperature_map:
+    variables:
+      tas: &map_settings
+        short_name: tas
+        start_year: 2081
+        end_year: 2100
+        mip: Amon
+        preprocessor: region_mean
+      tas_reference:
+        <<: *map_settings
```

Comment:
I think I'd avoid this yaml anchor as it only saves 1 line of code and just repeating it is more readable I think